src/plugins/range/CMakeLists.txt

```diff
@@ -1,5 +1,12 @@
 include (LibAddMacros)
 
+if (DEPENDENCY_PHASE)
+    set_source_files_properties(range.c PROPERTIES COMPILE_FLAGS -Wno-psabi)
+endif ()
+if(UNIX)
+    link_libraries(m)
```

Comment:
That is not a good idea, it adds the math library to *any* target added later.

Please use `LINK_LIBRARIES` in `add_plugin` instead.